src/agentlab2/envs/browser.py

```diff
@@ -0,0 +1,53 @@
+from agentlab2.benchmark import Task
+from agentlab2.core import Action, EnvironmentOutput
+from agentlab2.environment import EnvironmentConfig, ToolboxEnv
+from agentlab2.tools.playwright import SyncPlaywrightTool
+
+
+class BrowserEnvConfig(EnvironmentConfig):
+    """Configuration for BrowserEnv."""
+
+    headless: bool = True
+    timeout: int = 30000  # in milliseconds
+    use_html: bool = True
+    use_axtree: bool = False
+    use_screenshot: bool = True
+    prune_html: bool = True
+    pw_kwargs: dict = {}
+
+    def make(self, task: Task) -> "BrowserEnv":
+        """Create a BrowserEnv instance from the configuration for specified task."""
+        tool_config = self.model_dump()
+        tool_config.pop("pw_kwargs", None)
+        browser_tool = SyncPlaywrightTool(**tool_config, **self.pw_kwargs)
```

Comment:
Assuming it's temporary.
The BrowserEnvConfing should take a BrowserToolConfig that can instantiate the browser.